tests/test_frontend.py

```diff
@@ -175,37 +175,42 @@ def _tests(driver):
     # No hint button present
     assert not driver.find_elements_by_class_name("hint-icon")
 
-    # Skip forward to output prediction step
-    for _ in range(2):
+    # Skip forward to output prediction step: printing_the_range
+    for _ in range(3):
         skip_button.click()
         sleep(0.1)
 
     assert (
-        "range(n) is similar to the list"
+        "As you can see, the result is the same"
         in driver.find_element_by_css_selector(".book-text").text
     )
 
     # Correct answer first time
-    predict_output(driver, editor, run_button, 2, None)
+    predict_output(driver, editor, run_button, 0, None)
 
     # start again
     reverse_button.click()
 
     # Correct answer second time
-    predict_output(driver, editor, run_button, 1, 2)
+    predict_output(driver, editor, run_button, 1, 0)
 
     # start again
     reverse_button.click()
 
     # Two wrong answers
-    predict_output(driver, editor, run_button, 0, 1)
+    predict_output(driver, editor, run_button, 1, 2)
 
     # Click OK
     driver.find_element_by_css_selector(".submit-prediction button").click()
 
-    # Course has moved on to next step
+    # Course has moved on to next step: indices_out_of_bounds
+    # Let's skip to the step: index_exercise
+    for _ in range(6):  # change this back to 6
```

Comment:
change this back to 6?

Reply:
That's embarrassing :sweat_smile: I'll remove that.